hypertrace-graphql-label-application-rules-schema/src/main/java/org/hypertrace/graphql/label/application/rules/schema/shared/Action.java

```diff
@@ -0,0 +1,56 @@
+package org.hypertrace.graphql.label.application.rules.schema.shared;
+
+import graphql.annotations.annotationTypes.GraphQLField;
+import graphql.annotations.annotationTypes.GraphQLName;
+import graphql.annotations.annotationTypes.GraphQLNonNull;
+import java.util.List;
+import javax.annotation.Nullable;
+
+@GraphQLName(Action.TYPE_NAME)
+public interface Action {
+  String TYPE_NAME = "LabelApplicationAction";
+
+  String ENTITY_TYPES_KEY = "entityTypes";
+  String OPERATION_KEY = "operation";
+  String STATIC_LABELS = "staticLabels";
+  String DYNAMIC_LABEL_KEY_KEY = "dynamicLabelKey";
+  String VALUE_TYPE_KEY = "valueType";
+
+  @GraphQLName(Operation.TYPE_NAME)
+  enum Operation {
+    OPERATION_MERGE;
+    private static final String TYPE_NAME = "ActionOperator";
+  }
+
+  @GraphQLName(ValueType.TYPE_NAME)
+  enum ValueType {
+    STATIC_LABELS,
+    DYNAMIC_LABEL_KEY;
+    private static final String TYPE_NAME = "ValueType";
```

Comment:
Sorry, did not get you. Can you please elaboate?

Reply:
My bad - I was asking for a rename, because it's unclear where the word "Value" in the type name comes from in this file. This enum represents the type of `LabelApplicationAction`, so could be more clearly named `LabelApplicationActionType`.

Reply:
Done.